### PR TITLE
7pm-c Add scrollable view for messages

### PR DESCRIPTION
**Pull request by shouc**
We have added a scrollable view for the message display.

Before:

pr-64-before

After:

pr-64-after

### CODE REVIEW

**Review by M1Lucken — APPROVED** [review]

**Review by levypik — APPROVED** [review]

**Review by DanielEskander — APPROVED** [review]
lgtm

File: javascript/src/test/pages/Messages/SearchResults.test.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import userEvent from "@testing-library/user-event";
 //     };
 // });
```
Comment (pconrad) [inline javascript/src/test/pages/Messages/SearchResults.test.js:13]: I wonder whether this commented out code should be removed.

Is there a good reason why it is still here?  If not, then please remove it.  If so, let me know why.

File: javascript/src/test/pages/Messages/SearchResults.test.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ describe("ChannelPage tests", () => {
 //             'data': [exampleMessage]
 //         });
 
-//         const { getByText } = render(<ChannelPage />);
+//         const { getByText } = render(<ChannelPageList />);
```
Comment (pconrad) [inline javascript/src/test/pages/Messages/SearchResults.test.js:49]: I wonder whether this commented out code should be removed.

Is there a good reason why it is still here?  If not, then please remove it.  If so, let me know why.

**Review by pconrad — CHANGES_REQUESTED** [review]
I was about to type LGTM, when I saw a bunch of commented out code in one of your tests.

Commented out code should generally not be committed into the main branch unless there is a very good reason for it.   (Even if it was already in the code, that's not a good enough reason; you should generally remove it unless it serves a clear purpose.)

File: javascript/src/test/pages/Messages/SearchResults.test.js (hunk 2 above)
Comment (shouc) [reply javascript/src/test/pages/Messages/SearchResults.test.js]: I have replaced those comments with working test cases.

**Comment by pconrad** [comment]
40 pts!

**Review by pconrad — APPROVED** [review]

**Pull request merged**